.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -2,9 +2,9 @@
 <!-- Please perform the following checks and mark all the boxes accordingly. -->
 <!-- You can remove the checklist items that don't apply to your PR. -->
 
-- [ ] The page (if new), does not already exist in the repo.
+- [ ] The page (if new), does not already exist in the repository.
 - [ ] The page is in the correct platform directory (`common/`, `linux/`, etc.)
 - [ ] The page has 8 or fewer examples.
-- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
+- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message)
```

Comment:
Why?

Reply:
Thats odd.

I don't remember even being on that line so I don't have a clue how that went

Reply:
Oh, I see how it went.

My screen is the perfect size to leave it as a new line so it looks like it is a new line, which I got rid of because I thought I mispressed. Instead I removed it because I didn't even see it :rofl:

Reply:
https://github.com/tldr-pages/tldr/pull/6236/commits/201c2093a193e5666d149ee1cedf3a51078e8921